### DISCUSSION TITLE
Qfield Sync

### DISCUSSION THREAD

**Discussion by alexhogan18** [opengisch/QField#2713] [Q&A]
I am trying to use qfield cloud, I cant seem to download the experimental version of qfield sync even though I have enabled this in the settings, does anyone have any suggestions?

**Comment by ZsanettMed** [comment]
Hi @alexhogan18! There is no experimental version anymore. You can use the stable version v4.0.0 and connect to the cloud.

**Reply by alexhogan18** [reply]
Hi @ZsanettMed, is there a certain version of QGIS,  That I need to be using so that this version of Qfield Sync works, I am currently using version 3.8.3 and this was downloaded a couple of years ago so thinking maybe this is outdated?

**Reply by ZsanettMed** [reply]
Well, it´s quite old :) I would recommend updating to the latest version.

**Reply by suricactus** [reply]
@alexhogan18 please update to a supported version of QGIS and QFieldSync.
